### PR TITLE
JAVA-24414: Changes made for fixing failing test cases in core-java-st…

### PR DESCRIPTION
**Pull request by Bipinkumar27**
…rings

### CODE REVIEW

File: core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void whenDeclaringTooLongString_thenCompilationError() {
     
     @Test
     public void whenStoringInFileTooLongString_thenNoCompilationError() throws IOException {
```
Comment (anastasiosioannidis) [inline core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java:21]: Please revert this and just create a stringtoolong.txt in src/test/resources

File: core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java
Comment (anastasiosioannidis) [inline core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java:1]: In whenStoringInPropertiesString_thenNoCompilationError method:

Change "src/main/resources/config.properties" to:
"src/test/resources/config.properties"

and create the config.properties file in /test

File: core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java (hunk 1 above)
Comment (Bipinkumar27) [reply core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java:21]: sure

File: core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java
Comment (Bipinkumar27) [reply core-java-modules/core-java-strings/src/test/java/com/baeldung/stringtoolong/StringTooLongUnitTest.java:1]: sure

**Pull request merged**